applications/nrf5340_audio/src/bluetooth/bt_stream/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -315,37 +338,43 @@ int bt_le_audio_tx_send(struct bt_bap_stream **bap_streams, uint8_t *audio_mappi
 	return 0;
 }
 
-int bt_le_audio_tx_stream_stopped(uint8_t stream_idx)
+int bt_le_audio_tx_stream_stopped(struct stream_index stream_idx)
 {
 	if (!initialized) {
 		return -EACCES;
 	}
 
-	atomic_clear(&tx_info_arr[stream_idx].iso_tx_pool_alloc);
+	atomic_clear(&tx_info_arr[stream_idx.lvl1_idx][stream_idx.lvl2_idx][stream_idx.lvl3_idx]
+			      .iso_tx_pool_alloc);
 
 	return 0;
 }
 
-int bt_le_audio_tx_stream_started(uint8_t stream_idx)
+int bt_le_audio_tx_stream_started(struct stream_index stream_idx)
 {
 	if (!initialized) {
 		return -EACCES;
 	}
 
-	tx_info_arr[stream_idx].hci_wrn_printed = false;
-	tx_info_arr[stream_idx].iso_conn_handle = HANDLE_INVALID;
-	tx_info_arr[stream_idx].iso_tx.seq_num = 0;
-	tx_info_arr[stream_idx].iso_tx_readback.seq_num = 0;
+	uint8_t lvl1_idx = stream_idx.lvl1_idx;
```

Comment:
I prefer removing these temp variables. They only slightly shorten the lines.

Reply:
It shortens it by 33 characters per line, so that is quite significant, and when it is not shortened clang will make them go over multiple lines, making it harder to read.

Reply:
Change stream_idx -> idx, and with lvl1_idx -> lvl1 it will be as short, without extra variables